engine/pokemon/search.asm

```diff
@@ -46,6 +46,8 @@ MonCheck:
 	ret
 
 CheckOwnMonAnywhere:
+; BUG: CheckOwnMon does not check the Day-Care (see docs/bugs_and_glitches.md)
```

Comment:
```
; Check if the player owns any monsters of the species in wScriptVar.
; It must exist in either party or PC, and have the player's OT and ID.
; BUG: CheckOwnMon does not check the Day-Care (see docs/bugs_and_glitches.md)
```

Reply:
I moved the `; BUG:` comment slightly further down in this case.